pocs/state/states/default/pointing.py

```diff
@@ -82,6 +90,31 @@ def on_enter(event_data):
                 pocs.logger.debug("Pointing Coords: {}", pointing_image.pointing)
                 pocs.logger.debug("Pointing Error: {}", pointing_image.pointing_error)
 
+                separation = pointing_image.pointing_error.magnitude.value
+
+                # Correct the pointing
+                if should_correct and separation > pointing_threshold:
+                    pocs.say("I'm still a bit away from the field so I'm going to get closer.")
+
+                    # Tell the mount we are at the field, which is the center
+                    pocs.say("Syncing with the latest image...")
+                    has_field = pocs.observatory.mount.set_target_coordinates(pointing_image.pointing)
+                    pocs.logger.debug("Coords set, calibrating")
+
+                    # Sync the mount
```

Comment:
Definitely need a comment that explains what it means to sync the mount.

Reply:
Hoping to get that answer from iOptron. :) I'll add a basic comment.